modules/centralized-logging/variables.tf

```diff
@@ -40,11 +40,6 @@ variable "bucket_name" {
   type        = string
 }
 
-variable "parent_folder" {
```

Comment:
Hi @erlanderlo one thing we could do would be not to remove the variable now but add a deprecation notice in the description of the variable and making it optional, adding a default value `""` to it and not using it inside the module.

The deprecation notice could also explain that if the user has an existing deploy a new `apply` would move the audit log configuration to the projects instead of the folder.

This would impact only user how have deployed the `standalone` example
the private logging module is used only in this example as part of the harness.

Reply:
+1 thanks for considering this!  That helps and then we release a version with this notice and then a future release can fully remove it